### PR TITLE
Fix Resin issues

### PR DESCRIPTION
**Pull request by antonmolodykh**
Refs #674

### CODE REVIEW

File: Dockerfile.template [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM resin/%%RESIN_MACHINE_NAME%%-python
+FROM resin/%%RESIN_MACHINE_NAME%%-debian
 MAINTAINER Viktor Petersson <vpetersson@screenly.io>
```
Comment (vpetersson) [inline Dockerfile.template]: @antonmolodykh this should be you :)

Comment (antonmolodykh) [reply Dockerfile.template]: ok :)

File: Dockerfile.template [hunk 2]
```diff
@@ -21,38 +22,36 @@ RUN apt-get update && \
         x11-xserver-utils \
         xserver-xorg && \
     apt-get clean
+RUN apt-get -y remove stunnel4
```
Comment (vpetersson) [inline Dockerfile.template]: Why do we have this? If stunnel is installed already? Feels a bit redundant.

Comment (antonmolodykh) [reply Dockerfile.template]: it was in an ansible script, and I thought it was some kind of precaution

Comment (vpetersson) [reply Dockerfile.template]: Ah, yeah i think that was legacy. You should be able to remove that:

```
$ grep 'stunnel' ./* -R
./ansible/roles/ssl/tasks/main.yml:- name: Removes stunnel
./ansible/roles/ssl/tasks/main.yml:    name: stunnel4
```

**Comment by antonmolodykh** [comment]
Includes contributions from @imrehg

**Pull request merged**